terraform/modules/cloudwatch/main.tf

```diff
@@ -0,0 +1,19 @@
+locals {
```

Comment:
I wonder - if we're making everything module-based, why not use TF registry's modules instead?